share/templates/CMakeLists.txt

```diff
@@ -77,7 +77,10 @@ install(FILES
 
 install(FILES
       07-Band_and_Percussion/01-Concert_Band.mscz
-      07-Band_and_Percussion/02-Marching_Percussion.mscz
+      07-Band_and_Percussion/02-Marching_Band.mscz
+      07-Band_and_Percussion/03-Marching_Percussion_(battery).mscz
+      07-Band_and_Percussion/04-Marching_Percussion_(small_pit).mscz
+      07-Band_and_Percussion/05-Marching_Percussion_(large_pit).mscz
       DESTINATION ${Mscore_SHARE_NAME}${Mscore_INSTALL_NAME}templates/07-Band_and_Percussion
```

Comment:
I picked Battery_Percussion, Small_Pit, and Large_Pit.  The latter two names aren't perfect, since it's pit + battery, but I can't think of any better names that don't have parentheses.